src/os/bluestore/BlueStore.cc

```diff
@@ -12773,9 +12773,13 @@ void BlueStore::_do_omap_clear(TransContext *txc, const string& omap_prefix,
 			       uint64_t id)
 {
   string prefix, tail;
+  bufferlist tail_value;
   get_omap_header(id, &prefix);
   get_omap_tail(id, &tail);
   txc->t->rm_range_keys(omap_prefix, prefix, tail);
+  if (db->get(omap_prefix , tail, &tail_value) >= 0) {
```

Comment:
Do we really need this get() call? May be unconditionally call txc->t->rmkey(omap_prefix, tail) only?
Or even extend rm_range_keys() to support both inclusive and exclusive mode for tail parameter?

Reply:
fixed